packages/lesswrong/themes/siteThemes/eaTheme.ts

```diff
@@ -52,7 +55,7 @@ export const eaForumTheme: SiteThemeSpecification = {
       main: "bf360c",
     },
     background: {
-      default: '#f6f8f9'
+      default: '#f6f8f9' // TODO: why is this slightly different from LW? need to figure out what to do for dark mode
```

Comment:
@s-cheng because it's slightly blue shifted, due to my reading refactoring UI's advice "tint your greys".

Reply:
Yeah that makes sense, it does feel slightly nicer to me :) I'm not sure how to account for it in dark mode though.

Reply:
Probably we don't.

Reply:
I meant that, when I tested this branch, on dark mode our forum's background color was still light (on LW it properly turns dark).

Reply:
I see.